CONTRIBUTING.md

```diff
@@ -105,7 +105,7 @@ You can then go to `tmp/nx` (this is set up to use Nx CLI) or `tmp/angular` (thi
 
 ```bash
 yarn create-playground
-cd tmp/nx
+cd tmp/nx/proj
```

Comment:
@jaysoo while testing locally I noticed that the tests are using `proj`. Can you confirm, please? thanks.

Reply:
That's correct afaik. Probably an oversight in that document that it doesn't say that.